### DISCUSSION TITLE
JetBrains not working when refusing telemetry

### DISCUSSION THREAD

**Discussion by leoluk** [community/community#6949] [Copilot Conversations] [answered]
During login, the Jetbrains plugins prompts to accept telemetry, offering "OK" and "Cancel" buttons.

Pressing "Cancel" causes the login to succeed, but the plugin doesn't actually work and this warning appears in the log:

```
2021-10-28 23:12:54,550 [ 142531]   WARN - efaultCopilotCompletionService - fetchCompletions: telemetry terms were not accepted 
```

It should be made clear that refusing telemetry will disable the plugin. It's uncommon for something called "telemetry" to be required for core functionality.

**Comment by jansorg** [comment, marked as answer]
Thanks! The terms have to be accepted to get completions.
You're right that "OK" and "Cancel" are confusing, we'll change this to something better.
